install/common/build-image.py

```diff
@@ -64,111 +67,78 @@ def main():
 
     print(f"Building for architecture '{ARCH}'")
 
-    # Designate a unique tag for this image, depending on the Git commit it was created
-    # from:
-    # 1. If created from a Git tag (e.g., 0.8.0), the image tag will be `0.8.0`.
-    # 2. If created from a commit, it will be something like `0.8.0-31-g6bdaa7a`.
-    # 3. If the contents of the Git repo are dirty, we will append a unique identifier
-    #    for this run, something like `0.8.0-31-g6bdaa7a-fdcb` or `0.8.0-fdcb`.
-    dirty_ident = secrets.token_hex(2)
-    tag = (
-        subprocess.check_output(
-            ["git", "describe", "--long", "--first-parent", f"--dirty=-{dirty_ident}"],
+    if args.tag:
+        tag = args.tag
+    else:
+        # Designate a unique tag for this image, depending on the Git commit it was created
+        # from:
+        # 1. If created from a Git tag (e.g., 0.8.0), the image tag will be `0.8.0`.
+        # 2. If created from a commit, it will be something like `0.8.0-31-g6bdaa7a`.
+        # 3. If the contents of the Git repo are dirty, we will append a unique identifier
+        #    for this run, something like `0.8.0-31-g6bdaa7a-fdcb` or `0.8.0-fdcb`.
+        dirty_ident = secrets.token_hex(2)
+        tag = (
```

Comment:
That could go in its own function to ease readability.

Reply:
Implemented in 92d8a4c

Reply:
Thanks. I wonder if mentioning it's getting the tag name from git might help, since we might also have tags for the image.

Reply:
(in the name of the function, I mean)

Reply:
Hm, doesn't hurt. Check 0f0fa49923ef42d01c96b0e4163292f2f580279d.